csharp/Cli.cs

```diff
@@ -0,0 +1,203 @@
+        Console.WriteLine("  !delete - delete a specific song");
+        Console.WriteLine("  !stress - stress testing with mocked data");
+        Console.WriteLine("------------------------------------");
+    }
+
+    public async Task Start()
+    {
+        try
+        {
+            bool lifeTimeCli = true;
+            DisplayHint();
+        
+            while (lifeTimeCli)
+            {
+                var command = GetCommand();
+                switch (command)
+                {
+                    case "!add":
+                        await AddSong();
+                        break;
+                    case "!list":
+                        await ListSongs();
+                        break;
+                    case "!delete":
+                        await DeleteSong();
+                        break;
+                    case "!stress":
+                        await Stress();
+                        break;
+                    case "!q":
+                        await Exit();
+                        break;
+                    default:
+                        await Start();
+                        break;
+                }
+                DisplayHint();
+            }
+        }
+        catch(Exception ex)
+        {
+            Console.WriteLine(ex.Message);
+        }
+    }
+
+    private string GetCommand()
+    {
+        Console.Write("Type any command: ");
+        var command = Console.ReadLine();
+
+        Console.WriteLine("");
+        if (string.IsNullOrEmpty(command))
+        {
+            DisplayHint();
+            GetCommand();
+        }
+
+        return command;
+    }
+
+    private async Task AddSong()
+    {
+        Console.Write($"Song name: ");
+        string title = Console.ReadLine();
+        
+        Console.Write($"Album: ");
+        string album = Console.ReadLine();
+        
+        Console.Write($"Artist: ");
+        string artist = Console.ReadLine();
+        
+        var song = new Song()
+        {
+            Id = Guid.NewGuid(),
+            Album = album,
+            Artist = artist,
+            Title = title
+        };
+
+        Console.WriteLine($"Song {song.Title} from artist {song.Artist} Added!");
+
+        await _dataBase.Add(song);
+    }
+
+    private async Task ListSongs()
+    {
+        Console.WriteLine($"Here is the songs added so far: ");
+        Console.WriteLine($"-----------------------------------");
+        
+        var songs = await _dataBase.ListSongs();
+
+        ShowSongList(songs);
+        Console.WriteLine($"-----------------------------------");
+        Console.WriteLine("");
+    }
+
+    private async Task Stress()
+    {
+        var start = DateTime.Now;
+        Console.WriteLine("------------------------------------");
+        Console.WriteLine("Inserting 100.000 records into the database...");
+        Console.WriteLine(">    Starting...");
+        
+        var interation = 10000;
```

Comment:
The counter doesn't matches with the description above. 

```suggestion
        var interation = 100000;
```